src/module.ts

```diff
@@ -26,7 +26,11 @@ export default defineNuxtModule<PWAOptions>({
       display: 'standalone',
       background_color: '#ffffff',
       theme_color: '#000000',
-      icons: []
+      icons: [],
+      iosSplashSreen: {
```

Comment:
ultra simple options

based on my XP , more options could be great
exemple : 
special logo,
gradient bg

Reply:
For a first version I would had stick with Nuxt 2 PWA module behavior which hasn't `iosSplashScreen` option but only uses `mobileAppIOS` option to enable the splash screen features.

Maybe the splash background color is already important for your projects ?

Reply:
fine I will change and stick to it
It will be for next release